CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * [#205](https://github.com/mongoid/mongoid-history/pull/205): Allow modifier field to be optional - [@yads](https://github.com/yads).
 * [#211](https://github.com/mongoid/mongoid-history/pull/211): Enable tracking create/destroy by default - [@jagdeepsingh](https://github.com/jagdeepsingh).
 * [#212](https://github.com/mongoid/mongoid-history/pull/212): `track_history` method support for `:if` and `:unless` options - [@jagdeepsingh](https://github.com/jagdeepsingh).
+* [#213](https://github.com/mongoid/mongoid-history/pull/213): Skip yielding when no block provided - [@mikwat](https://github.com/mikwat).
```

Comment:
It's unclear what is no longer yielding, so should say `Do not yield on track_history when no block is provided` or something like that.